eZ/Publish/Core/FieldType/Tests/SelectionTest.php

```diff
@@ -46,7 +46,7 @@ protected function createFieldTypeUnderTest()
      */
     protected function getValidatorConfigurationSchemaExpectation()
     {
-        return array();
+        return [];
```

Comment:
It's ok'ish for this PR now that you have already done it, but please don't do unrelated changes on PR's, especially not on parts not otherwise changed in the PR. It makes it really hard to review the changes.

FYI plan with @alongosz is to at some point update php-cs-fixer version, and apply some updates to our CS like this one and for instance [no_superfluous_phpdoc_tags](https://twitter.com/Stof70/status/1019515955083403264), and apply that on 6.7 and up in bulk while most people are on vacation, so for instance august.

/cc @ezsystems/support-team

Reply:
Look at this again, could we drop the unrelated array changes @mateuszbieniek ?